app/views/videos/show.html.haml

```diff
@@ -7,6 +7,37 @@
         %header
           %h3= @video.title
           %h4=link_to @video.category.name, category_path(@video.category)
```

Comment:
I would use shorthand routes - `@video.category` instead of `category_path(@video.category)`.